Patch from https://github.com/mehcode/python-xmlsec/issues/84

Deals with test suite failure due to LXML's tendency to
reset the global parser state.
--- a/tests/base.py
+++ b/tests/base.py
@@ -99,6 +99,7 @@
 
     def load_xml(self, name, xpath=None):
         """returns xml.etree"""
+        etree.set_default_parser(parser=etree.XMLParser())
         root = etree.parse(self.path(name)).getroot()
         if xpath is None:
             return root
--- a/tests/test_doc_examples.py
+++ b/tests/test_doc_examples.py
@@ -42,3 +42,5 @@
     """
     with cd(example.parent):
         runpy.run_path(str(example))
+    from lxml import etree
+    etree.set_default_parser(parser=etree.XMLParser())
